src/containers/About/About.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import './style.css';
+import './style.scss';
 import { ContributorItem } from '../../components/Contributor/ContributorItem';
 
 function About() {
   return (
-    <div className="about__container">
-      <h2>about us</h2>
```

Comment:
Since we decided to use scss. Could use className for h2 tag. e.g "About__title".